routes/game.rb

```diff
@@ -145,7 +145,19 @@ class Api
               halt(400, 'Player count not supported')
             end
 
-            set_game_state(game, engine, users)
+            acting = set_game_state(game, engine, users)
+
+            unless acting.empty?
```

Comment:
why would this ever be empty?

Reply:
I don't know. Just copied what was above. But indeed it seems highly unlikely that nobody is acting when the games is starting.